src/engine/input/key-mapping/InputActionItem.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using Godot;
+using Newtonsoft.Json;
+
+public class InputActionItem : VBoxContainer
+{
+    [Export]
+    public NodePath AddInputEventPath;
+
+    [Export]
+    public NodePath InputActionHeaderPath;
+
+    [Export]
+    public NodePath InputEventsContainerPath;
+
+    internal InputGroupItem AssociatedGroup;
+
+    private Label inputActionHeader;
+    private HBoxContainer inputEventsContainer;
+    private Button addInputEvent;
+
+    [JsonProperty]
```

Comment:
As this is a GUI element in the options menu, this doesn't need saving, right?
Why did you add these attributes here?